benchmarks/movielens.ipynb

```diff
@@ -0,0 +1,942 @@
+{
```

Comment:
Maybe hyperparater tuning can be incorporated given that we have already developed notebooks for some (unfortunately not all) algorithms.

Reply:
Just saw @anargyri comment 😂

Reply:
not sure if you guys mean to have the tuning in this same notebook or to perform the tuning outside and then add the optimal parameters here?

Reply:
The former requires writing a python script for each method. It is a lot of work and will take a long time to run. Probably the latter is more reasonable but can be done gradually, not all of it in this PR. You could start from ALS and SVD since we have automated tuning for those.

Reply:
that makes sense